docs/src/use/migrate-to-9.0.0.md

```diff
@@ -55,6 +56,14 @@ ESLint is officially dropping support for these versions of Node.js starting wit
 
 **Related issue(s):** [#17595](https://github.com/eslint/eslint/issues/17595)
 
+## <a name="flat-config"></a> New config format (`eslint.config.js`)
+
+In ESLint v9.0.0, [`eslint.config.js`](./configure/configuration-files) is the default configuration format.
+
+**To address:** Update your configuration to the new format following the [Configuration Migration Guide](./configure/migration-guide). In case you still need to use the old `eslintrc` config format, set environment variable `ESLINT_USE_FLAT_CONFIG` to `false`.
```

Comment:
```suggestion
**To address:** Update your configuration to the new format following the [Configuration Migration Guide](./configure/migration-guide). In case you still need to use the deprecated eslintrc config format, set environment variable `ESLINT_USE_FLAT_CONFIG` to `false`.
```